public/app/features/dashboard-scene/saving/SaveProvisionedDashboard.tsx

```diff
@@ -43,7 +51,13 @@ export interface Props {
 
 export function SaveProvisionedDashboard({ meta, drawer, changeInfo, dashboard }: Props) {
   const [saveDashboard, request] = useUpdateRepositoryFilesMutation();
-  const { register, handleSubmit } = useForm({ defaultValues: getDefaultValues(meta) });
+  const defaultValues = getDefaultValues(meta);
+  const { register, handleSubmit, watch } = useForm({ defaultValues });
+  const repositoryConfigQuery = useGetRepositoryQuery({ name: defaultValues.repo });
+  const repositoryConfig = repositoryConfigQuery?.data?.spec;
+  const isGitHub = repositoryConfig?.type === 'github';
+  const [repo, ref] = watch(['repo', 'ref']);
+  const href = createPRLink(repositoryConfig, repo, ref);
```

Comment:
rather just showing a link; I think this needs to be what happens *after* successfully writing to the branch.

Reply:
You mean do not show the link until the branch has some commits?